docs/reference/mapping/runtime.asciidoc

```diff
@@ -188,6 +164,26 @@ PUT my-index/
 }
 ----
 
+When no script is provided, {es} implicitly looks in `_source` at query time
+for a field with the same name as the runtime field, and returns a value if one
+exists. If a field with the same name doesn’t exist, the response doesn't
+include any values for that runtime field.
+
+In most cases, you should retrieve field values through
+<<doc-values,`doc_values`>> whenever possible. Accessing `doc_values` with a
+runtime field is faster than retrieving values from `_source` because of how
+data is loaded from Lucene.
+
+However, there are cases such as `text` fields where `doc_values` are not
+available by default, or where you choose to not index specific fields or set
```

Comment:
or where you choose to disable `doc_values`. Indexing here can be confusing: you may have a mapped field with index:true and doc_value: false. That field is not indexed and you need to load it from _source if you want to access its values from a script. Yet the field is searchable, hence you don't need a runtime field to be able to query it, but it is not aggregatable.

Reply:
Thanks @javanna, I'll clarify that use case.